packages/data/README.md

```diff
@@ -286,6 +286,93 @@ const SaleButton = withDispatch( ( dispatch, ownProps ) => {
 //  <SaleButton>Start Sale!</SaleButton>
 ```
 
+## Generic Stores
+
+The `@wordpress/data` module offers a more advanced and generic interface for the purposes of integrating other data systems and situations where more direct control over a data system is needed. In this case, a data store will need to be implemented outside of `@wordpress/data` and then plugged in via three functions:
+
+- `getSelectors()`: Returns an object of selector functions, pre-mapped to the store.
+- `getActions()`: Returns an object of action functions, pre-mapped to the store.
+- `subscribe( listener: Function )`: Registers a function called any time the value of state changes.
+   - Behaves as Redux [`subscribe`](https://redux.js.org/api-reference/store#subscribe(listener))
+   with the following differences:
+      - Doesn't have to implement an unsubscribe, since the registry never uses it.
+	  - Only has to support one listener (the registry).
+
+By implementing the above interface for your custom store, you gain the benefits of using the registry and the `withSelect` and `withDispatch` higher order components in your application code. This provides seamless integration with existing and alternative data systems.
+
+Integrating an existing redux store with its own reducers, store enhancers and middleware can be accomplished as follows:
+
+_Example:_
+
+```js
+import existingSelectors from './existing-app/selectors';
+import existingActions from './existing-app/actions';
+import createStore from './existing-app/store';
+
+const reduxStore = createStore();
+
+const mappedSelectors = existingSelectors.map( ( selector ) => {
+	return ( ...args ) => selector( reduxStore.getState(), ...args );
+} );
+
+const mappedActions = existingActions.map( ( action ) => {
+	return actions.map( ( action ) => {
+		return ( ...args ) => reduxStore.dispatch( action( ...args ) );
+	} );
+} );
+
+const genericStore = {
+	getSelectors(): {
```

Comment:
You can remove the colon. Not valid js, right? Also in examples below

Reply:
Yes, thanks again.